files/bootstrap.switch/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "bootstrap.switch",
+  "repo": "nostalgiaz/bootstrap-switch",
+  "files": {
+    "include": ["./dist/**/*"]
```

Comment:
This rule will create a `dist` directory which is wrong.
Instead use the `basePath` parameter to set `dist` and then you can remove `include` completely